environment/src/main/java/com/sequenceiq/environment/environment/v1/converter/NetworkRequestToDtoConverter.java

```diff
@@ -31,6 +33,9 @@ public class NetworkRequestToDtoConverter {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(NetworkRequestToDtoConverter.class);
 
+    @Value("${cb.multiaz.azure.availabililtyZones:1,2,3}")
```

Comment:
please move default values to application yaml

Reply:
code is already handling to read it from application.yml. I have also put a default value while reading it if it is not specified. In future, it can be added to application.yml and will override the value. Do you want me to add this value in application.yml as well?

Reply:
have the default value in the app yaml and remove from here. we usually avoid setting defaults in the code. you might do this if the class is included in multiple services  so you would have to add it multiple times.
```suggestion
    @Value("${cb.multiaz.azure.availabililtyZones}")
```

Reply:
Done